compiler/frontend/spark/src/main/java/org/apache/nemo/compiler/frontend/spark/transform/MapToPairTransform.java

```diff
@@ -64,6 +65,11 @@ public void onWatermark(final Watermark watermark) {
     outputCollector.emitWatermark(watermark);
   }
 
+  @Override
+  public void onLatencymark(final Latencymark latencymark) {
```

Comment:
Lots of duplicate onLatencyMark code blocks. It would be good to refactor it.

Reply:
done